java-squid/src/main/java/org/sonar/java/xml/XPathXmlCheck.java

```diff
@@ -19,12 +19,11 @@
  */
 package org.sonar.java.xml;
 
-import org.sonar.squidbridge.api.AnalysisException;
 import org.w3c.dom.Node;
 
 import javax.xml.xpath.XPathExpressionException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.xml.xpath.XPathExpressionException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)